src/app/Shared/Redux/ReduxStore.tsx

```diff
@@ -60,3 +62,4 @@ export type Store = ReturnType<typeof setupStore>;
 // Add a subscription to save filter states to local storage
 // if states change.
 store.subscribe(() => saveToLocalStorage('TARGET_RECORDING_FILTERS', store.getState().recordingFilters.list));
+store.subscribe(() => saveToLocalStorage('AUTOMATED_ANALYSIS_FILTERS', store.getState().automatedAnalysisFilters.list));
```

Comment:
I think the callback to `store.subscribe` will be called if anywhere in the state tree changes (i.e. both will be called regardless of which filters changed). So, I think we can group this in a single `store.subscribe`.